site/content/en/docs/Examples/_index.md

```diff
@@ -21,8 +21,6 @@ These are full examples for each of the resource types of Agones
 
 These are all examples of simple game server implementations, that integrate the Agones game server SDK. 
 
-- {{< ghlink href="examples/simple-udp" >}}Simple UDP{{< /ghlink >}} (Go) - simple server and client that send UDP packets back and forth.
```

Comment:
Should this be deleted now or with the next release?

Reply:
So I didn't see any specific reason not to delete it now, given that it is a duplicated example that has almost identical code already stored in the `simple-game-server`. But if you feel strongly we should keep it until next release, I'm happy to do so.

Reply:
I think this was the main place I wanted to gate the change. In the rest of the site it seems ok to flip it over, but removing these links separate from a release (where we remove support for the code) didn't feel quite right.